### ISSUE TITLE
Incorrect JSON-LD encoding of provenance fields in model and artifacts

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-model#1]
In CEDAR resources, the `schema:isBasedOn`, `pav:createdOn`, `pav:createdBy`, `pav:lastUpdatedOn`, and `oslc:modifiedBy` fields are effectively represented as strings.

For example:

```
{
   "schema:isBasedOn": "https://repo.metadatacenter.org/template/4343",
   "pav:createdOn": "2016-06-29T10:58:26-0700",
   "pav:createdBy":  "https://repo.metadatacenter.net/users/6d21a887",
   "pav:lastUpdatedOn": "2016-06-29T10:58:26-0700",
    "oslc:modifiedBy": "https://repo.metadatacenter.net/users/6d21a887",
   ...
}
```

However, the `schema:isBasedOn`, `pav:createdBy`,  and `oslc:modifiedBy` fields should be represented as URIs and the `pav:createdOn` and `pav:lastUpdatedOn` fields should be represented as `xsd:dateTime` fields.

We can do this by expanding the current `@context` field to include:

```
@context {
  ...
  "xsd": "http://www.w3.org/2001/XMLSchema#",
  "schema:isBasedOn": { "@type": "@id" },
  "pav:createdBy": { "@type": "@id" }
  "oslc:modifiedBy": { "@type": "@id" }
  "pav:createdOn": {  "@type": "xsd:dateTime" },
  "pav:lastUpdatedOn": {  "@type": "xsd:dateTime" }
}
```

**Comment by martinjoconnor** [comment]
See also:  metadatacenter/cedar-template-editor#531

**Comment by martinjoconnor** [comment]
Merged into metadatacenter/cedar-project#185
